runner.yml.dist

```
@@ -14,75 +14,67 @@ drupal:
+    - { task: "exec", command: "./vendor/bin/drush en oe_search oe_corporate_blocks oe_corporate_site_info field_ui config_devel toolbar -y" }
+    - { task: "exec", command: "./vendor/bin/drush en oe_multilingual -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set system.theme default oe_theme -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set system.theme admin claro -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set user.settings register visitors -y" }
+    - { task: "exec", command: "./vendor/bin/drush cr" }
+  additional_settings: |
+        if (isset($config['system.performance']['cache']['page']['max_age'])) {
+          unset($config['system.performance']['cache']['page']['max_age']);
+        }
+        $settings['file_scan_ignore_directories'] = array('vendor', 'build', 'node_modules', 'bower_components');
+        $settings['file_private_path'] = 'sites/default/files/private';
+        $settings['extension_discovery_scan_tests'] = TRUE;
+        $databases['sparql_default']['default'] = [
+          'host' => 'sparql',
+          'port' => '8890',
+          'namespace' => 'Drupal\sparql_entity_storage\Driver\Database\sparql',
+          'driver' => 'sparql',
+        ];
+
 selenium:
   host: "http://selenium"
   port: "4444"
   browser: "chrome"
 
 commands:
   drupal:site-setup:
+    - { task: "exec", command: "bash ./scripts/build-ecl-pipeline.sh" }
     - { task: "run", command: "drupal:symlink-project" }
     - { task: "symlink", from: "${drupal.root}/core/.stylelintrc.json", to: ".stylelintrc.json" }
     - { task: "run", command: "drupal:drush-setup" }
     - { task: "run", command: "drupal:settings-setup" }
-    # Generate settings.testing.php, it will be used when running functional tests.
-    - { task: "process-php", type: "write", config: "drupal.settings", source: "${drupal.root}/sites/default/default.settings.php", destination: "${drupal.root}/sites/default/settings.testing.php", override: true }
     - { task: "run", command: "setup:phpunit" }
     - { task: "run", command: "setup:behat" }
   setup:phpunit:
     - { task: "process", source: "phpunit.xml.dist", destination: "phpunit.xml" }
+    - { task: "remove", file: "build/sites/default/settings.testing.php" }
+    - task: "append"
+      file: "build/sites/default/settings.testing.php"
+      text: |
+          <?php
+              $settings['extension_discovery_scan_tests'] = TRUE;
+              $databases['sparql_default']['default'] = [
+                'host' => 'sparql',
+                'port' => '8890',
+                'namespace' => 'Drupal\sparql_entity_storage\Driver\Database\sparql',
+                'driver' => 'sparql',
+              ];
```

```suggestion
      text: |
        <?php

        $settings['extension_discovery_scan_tests'] = TRUE;
        $databases['sparql_default']['default'] = [
          'host' => 'sparql',
          'port' => '8890',
          'namespace' => 'Drupal\sparql_entity_storage\Driver\Database\sparql',
          'driver' => 'sparql',
        ];
```
This would make more sense to me.
Or is there a special reason I don't know about?